extensions/canvascast/src/index.tsx

```diff
@@ -100,47 +148,52 @@ export default function main() {
                       }))
                     );
                   })
-                  .catch(() => {
-                    // ignore error?
+                  .catch((err: any) => {
+                    showToast(ToastStyle.Failure, `Error: ${err.message}`);
                   });
+                setLoading(false);
               })
-              .catch(() => {
-                // ignore error?
+              .catch((err: any) => {
+                showToast(ToastStyle.Failure, `Error: ${err.message}`);
               });
           })
-          .catch(() => {
-            // ignore error?
+          .catch((err: any) => {
+            showToast(ToastStyle.Failure, `Error: ${err.message}`);
           });
       })
-      .catch(() => {
-        // ignore error?
+      .catch((err: any) => {
+        showToast(ToastStyle.Failure, `Error: ${err.message}`);
       });
-  });
+  }, []);
 
   return (
-    <List isLoading={items === undefined}>
-      <List.Section title="Courses" subtitle="Your enrolled courses">
+    <List isLoading={loading}>
+      <List.Section title="Courses">
         {items?.map((item, index) => (
           <List.Item
             key={index}
             title={item.name}
-            subtitle={item.code}
-            icon={{ source: Icon.Pin, tintColor: Color[Colors[index % Colors.length]] }}
+            icon={{ source: Icons["Course"], tintColor: Color[Colors[index % Colors.length]] }}
             actions={
-              <ActionPanel title="Title">
-                <OpenInBrowserAction url={"https://" + preferences.domain + "/courses/" + item.id} />
```

Comment:
I love this new update. Thanks for making the extension better 😄!

I do have some small feedback: It was really convenient to be able to easily visit the home page of courses. It seems like this functionality was replaced with the modules (which is totally fine!). Would it be possible to keep the link to course home pages?